Makefile

```diff
@@ -34,7 +34,7 @@ linter: gotools build
 
 gotools:
 	# install goimports if not present
-	$(GO) install golang.org/x/tools/cmd/goimports
+	cd tools && $(GO) install golang.org/x/tools/cmd/goimports
```

Comment:
wat?

there isn't a tools directory.

Reply:
uhhh this one is tricky ... I moved tools.go to tools as the fabric linter does not work when go files are in the root dir.

Reply:
Below it is added as tools.go is moved. But do we really need a new directory `tools`, if we don't want it on the top-level (for which i see some argument), why not move it to `utils`?

Reply:
I dont have a strong opinion if we move this to tools/, utils/, or even utils/tools. I guess if we move it to a dedicated tool folder we can fix tool dependencies easily. 

BTW fabric also has it in tools/, see here https://github.com/hyperledger/fabric/tree/master/tools

Reply:
what's the difference between utils and tools?  all stuff in utils are tools?  So i don't see the reason to have an additional directory....

Reply:
moved to utils